app/.env.development

```diff
@@ -3,6 +3,7 @@
 # env vars when running `next dev` only.
 # Learn more: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
 
+NODE_ENV=development
```

Comment:
Just clarifying, the aliases and reloading weren't working without this? If so, may be helpful to include code a comment above this with that context. For some reason I thought this was automatically set when running in dev mode.

Reply:
More on that here: https://github.com/navapbc/template-application-nextjs/pull/300#discussion_r1486762108

It just seems like "good practice" at this point since alias resolution is now working in docker.